### PR TITLE
Allow creating alerts with multiple notification channels

### PR DESCRIPTION
**Pull request by priyawadhwa**
This way we can send alerts to PD and Slack

Signed-off-by: Priya Wadhwa <priya@chainguard.dev>

### CODE REVIEW

File: terraform/gcp/modules/monitoring/dex/variables.tf [hunk 1]
```diff
@@ -44,10 +44,10 @@ variable "dex_url" {
 
 // Set-up for notification channel for alerting
 variable "notification_channel_id" {
```
Comment (k4leung4) [inline terraform/gcp/modules/monitoring/dex/variables.tf:46]: not sure if it is worthwhile to do or not, but rename to "notification_channel_ids"

and updating the description to indicate it takes more than one id?

same in other modules

Comment (priyawadhwa) [reply terraform/gcp/modules/monitoring/dex/variables.tf]: done

**Review by k4leung4 — APPROVED** [review]
thanks

**Comment by priyawadhwa** [comment]
@k4leung4 sorry had a lint error, could you re-approve?? thanks!!

**Review by k4leung4 — APPROVED** [review]

**Pull request merged**